Ghidra/Features/Base/src/main/java/ghidra/app/util/bin/format/omf/OmfSymbol.java

```diff
@@ -26,6 +26,14 @@ public class OmfSymbol {
 	private long offset;
 	private Address address;
 
+	private final String FloatingPointSpecialNames[] = {
```

Comment:
/* need to add docs about where these symbol names were found and their significance */
private  static final Set<String> FLOATINGPOINT_SPECIALNAMES = Set.of("FIWRQQ", "FIDRQQ"... "FJGRQQ");